internal/utils/utils.go

```diff
@@ -279,3 +280,9 @@ func SortHPAMetricsSpec(metrics []autoscalingv2.MetricSpec) []autoscalingv2.Metr
 	})
 	return metrics
 }
+
+// CalculateHash calculates a SHA256 hash for the given data
+func CalculateHash(data string) string {
```

Comment:
nit: rename to CalculateSHA256 to be more precise

Reply:
done